### PR TITLE
Update Trace-flags with 9810

### PR DESCRIPTION
**Pull request by PiJoCoder**
#hold-off

### CODE REVIEW

File: docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md [hunk 1]
```diff
@@ -234,6 +234,7 @@ Trace flag behavior may not be supported in future releases of [!INCLUDE[ssNoVer
...
 | <a id="tf9708"></a>**9708** | Enables collection of event publishing metrics for extended event sessions. For more information, see [sys.dm_xe_session_events](../../relational-databases/system-dynamic-management-views/sys-dm-xe-session-events-transact-sql.md).<br /><br />**Applies to:** [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)] and later versions.<br /><br />**Scope:** Global only. |
+| <a id="tf9810"></a>**9810** | Disables In-Memory OLTP enging from reclaiming Thread Local Storage (TLS) memory. Not reclaiming TLS memory is the default behavior in SQL Server 2019 and earlier version. In SQL Server 2022, a new memory optimization was introduced which causes Hekaton engine to reclaim TLS memory and to reduce possibily of out-of-memory issues. This trace flag reverts to the behavior prior to SQL Server 2022. For more information, see .<br /><br />**Applies to:** [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)] and later versions.<br /><br />**Scope:** Global only. |
```
Comment (rwestMSFT) [inline docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md]: Workshopping the main content. I see there's a "more info" link still missing, but I hope this helps in the meantime.

```suggestion
| <a id="tf9810"></a>**9810** | Disables the In-Memory OLTP engine from reclaiming Thread Local Storage (TLS) memory. In [!INCLUDE[sssql19-md](../../includes/sssql19-md.md)] and earlier versions, not reclaiming TLS memory is the default behavior. In [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)], a new memory optimization was introduced that causes the In-Memory OLTP engine to reclaim TLS memory, and to reduce the possibility of out-of-memory issues. This Trace Flag reverts to the behavior prior to [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)]. For more information, see .<br /><br />**Applies to:** [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)] and later versions.<br /><br />**Scope:** Global only. |
```

**Comment by prmadhes-msft** [comment]
@PiJoCoder Below details can be add as well in trace flag table 

**9810**   In-memory OLTP management  - Improve memory management in large memory servers to reduce out-of-memory conditions.
**Applies to**: SQL Server 2022 (16.x) and later versions.

**Scope**: Global only.

**Comment by PiJoCoder** [comment]
> @PiJoCoder Below details can be add as well in trace flag table
> 
> **9810** In-memory OLTP management - Improve memory management in large memory servers to reduce out-of-memory conditions. **Applies to**: SQL Server 2022 (16.x) and later versions.
> 
> **Scope**: Global only.

Pradeep, the info you provided is the what got added in 2022. This trace flag reverts this behavior and essentially enables the 2019 behavior.

**Comment by prmadhes-msft** [comment]
> > @PiJoCoder Below details can be add as well in trace flag table
> > **9810** In-memory OLTP management - Improve memory management in large memory servers to reduce out-of-memory conditions. **Applies to**: SQL Server 2022 (16.x) and later versions.
> > **Scope**: Global only.
> 
> Pradeep, the info you provided is the what got added in 2022. This trace flag reverts this behavior and essentially enables the 2019 behavior.

Ok, Got it.

**Pull request closed without merging**